tutorials/llm/llama-3/README.md

```diff
@@ -0,0 +1,145 @@
+    *  Access to atleast 1 NVIDIA GPU with a cumulative memory of atleast 80GB, for example: 1 x H100-80GB or 1 x A100-80GB.
+    * A docker enabled environment, with [NVIDIA Container Runtime](https://developer.nvidia.com/container-runtime) installed, which will make the container GPU-aware.
+    * Other pre-requisites for NIM are mentioned [here](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#prerequisites)
+    
+* Requested the necessary permission from Hugging Face and Meta to download [Meta-Llama-3-8B-Instruct](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct). Then, you can use your Hugging Face [access token](https://huggingface.co/docs/hub/en/security-tokens) to download the model, which we will then convert and customize with NeMo Framework.
+
+* [Authenticate with NVIDIA NGC](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#ngc-authentication), and download [NGC CLI Tool](https://docs.nvidia.com/nim/large-language-models/latest/getting-started.html#ngc-cli-tool).
+
+
+
+
+## [1. Creating a LoRA adapter with NeMo Framework](./llama3-lora-nemofw.ipynb)
+
+This notebook showcases performing LoRA PEFT **Llama-3 8B** on [PubMedQA](https://pubmedqa.github.io/) using NeMo Framework. PubMedQA is a Question-Answering dataset for biomedical texts.
+
+
+### To get started
+
+NeMo Framework is available as a [docker container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo). 
+
+Run the container using the following command. It assumes that you have the notebook(s) available in the current working directory. If not, mount the appropriate folder to `/workspace`.
+
+```
+export FW_VERSION=24.05 ## Make sure to choose the latest available tag
+
+docker run \
+  --gpus all \
+  --shm-size=2g \
+  --net=host \
+  --ulimit memlock=-1 \
+  --rm -it \
+  -v ${PWD}:/workspace \
+  -w /workspace \
+  -v ${PWD}/results:/results \
+  nvcr.io/nvidia/nemo:$FW_VERSION bash
+```
+
+From within the container, start Jupyter lab:
+
+```
+jupyter lab --ip 0.0.0.0 --port=8888 --allow-root
+```
+Then, navigate to [this notebook](./llama3-lora-nemofw.ipynb).
+
+
+## [2. Multi-LoRA inference with NVIDIA NIM](./llama3-lora-deploy-nim.ipynb)
+This is a demonstration of deploying multiple LoRA adapters with NVIDIA NIM. NIM supports LoRA adapters in .nemo (from NeMo Framework), and Hugging Face model formats. We will deploy the PubMedQA LoRA adapter from Notebook 1, alongside two other already trained LoRA adapters ([GSM8K](https://github.com/openai/grade-school-math), [SQuAD](https://rajpurkar.github.io/SQuAD-explorer/)) that are available on NVIDIA NGC as examples.
+
+`NOTE`: While it's not necessary to complete the LoRA training and obtain the adapter from the previous notebook ("Creating a LoRA adapter with NeMo Framework") to follow along with this one, it is recommended if possible. You can still learn about LoRA deployment with NIM using the other adapters downloaded from NGC.
```

Comment:
Could use a blockquote for a note as well.

Reply:
fixed

Reply:
This says "fixed" but it's not currently in blockquote format - intentional?